modules/installation-custom-aws-vpc.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-vpc.adoc
+
+[id="installation-custom-aws-vpc_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into existing subnets in an existing VPC in Amazon Web Services (AWS). By deploying {product-title} into an existing AWS VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. If you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself, use this installation option.
```

Comment:
@bergerhoffer, I want to keep both because you're trying to avoid the fact that your default, or carefully curated corporate, account is limited. (You usually don't have enough space in your account to make all the cluster components, and it's apparently common for cloud admins to not let you make networking components.)

Would "avoid service limit constraints" scan better?

Reply:
@kalexand-rh oh i see, i didn't realize that "limit constraints" was what it actually is there. It sounded like a double word to me. If it works for you, "avoid service limit constraints" would be much clearer to me.

Reply:
It does indeed! Thank you for pointing this out. I'll update all of the VPC PRs. :)